micah-akpan/bucketlist-api
# Feature: Create API server

#### What does this PR do?
This PR sets up the API server that services request from the client
#### Description of Task to be completed?
- Builds API server
#### How should this be manually tested?
- Git fetch and checkout to `feat/build-api-server`
- Install Deps: `yarn start`
- Start server: `yarn dev`
- Make an API request using any HTTP client to `http://localhost:2020`
- You should get a welcome response in JSON format
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168608803](https://www.pivotaltracker.com/story/show/168608803)
#### Screenshots (if appropriate)
